tests/Entity/SessionTest.php

```diff
@@ -77,7 +77,19 @@ public function testConstructor()
      */
     public function testSetTitle()
     {
-        $this->basicSetTest('title', 'string');
```

Comment:
Something hinkey here, maybe the title test got overwritten?

Reply:
copy/paste error. fixed.